### DISCUSSION TITLE
Mixed Content

### DISCUSSION THREAD

**Discussion by Creveoolus** [miguelgrinberg/Flask-SocketIO#1668] [Q&A]
Mixed Content: The page at 'https://domain.com/chat' was loaded over HTTPS, but requested an insecure XMLHttpRequest endpoint 'http://domain.com/socket.io/?EIO=3&transport=polling&t=NjUu7RG'. This request has been blocked; the content must be served over HTTPS.

Is this Flask-SocketIO related? If yes, how can i fix it?

**Comment by miguelgrinberg** [comment]
Please don't open issues when you have a question. Didn't you see that there is a discussion board? That's the place for questions.

The problem here is in your Socket.IO client, which is using a bad connection URL. It should use an https:// URL, but it is using an http:// URL instead.
